.travis.yml

```diff
@@ -0,0 +1,9 @@
+sudo: required
+dist: trusty
+language: python
+install:
+  - sudo apt-get -y install gdb
```

Comment:
I suppose, but any custom version ought to be installed at not-/usr/bin/gdb (e.g. .../local/gdb)